src/library-authoring/create-library/LibraryCreatePage.jsx

```diff
@@ -187,6 +198,43 @@ export class LibraryCreatePage extends React.Component {
     }
   }
 
+  validateInput(fieldName, value, data) {
+    const { intl } = this.props;
+    const { errors } = this.state;
+    const slugRegex = new RegExp(VALID_SLUG_ID_REGEX);
+
+    switch (fieldName) {
+      case 'org':
+        if (!value) {
+          errors[fieldName] = intl.formatMessage(messages['library.form.field.error.empty']);
+        } else if (value && !data.org) {
+          errors[fieldName] = intl.formatMessage(messages['library.form.field.error.mismatch.org']);
+        } else {
+          errors[fieldName] = '';
```

Comment:
Is this correct that `errors[fieldName] = '';` adds error for the field and in the input props `hasFieldError={this.hasFieldError('org')}` will always be `true`?

Reply:
removed {this.hasFieldError('org')} props because it unused